src/index.ts

```diff
@@ -40,7 +44,25 @@ const useAbortableFetch = <T>(
               status: rsp.status
             })
       )
-      .then(rsp => rsp.json())
+      .then(rsp => {
+        if (!init.streaming) return rsp.json();
+        if (!rsp.body) throw new Error('Invalid response body');
+
+        const reader = rsp.body.getReader();
+        (async () => {
+          while (true) {
+            const { value, done } = await reader.read();
+            if (done) {
+              break;
+            }
+
+            setState((oldState: FetchState) => ({
+              ...oldState,
+              data: value,
```

Comment:
Should this create a new Uint8Array with the already received chunks and the new one appended?

Reply:
I tried that first but it actually kept on re-instantiating the state so I only got the latest log. For my use-case, I handled the aggregation from the caller of the hooks instead.

Reply:
The `oldState` contains the `data` loaded so far.

Reply:
👍 got it working now

Reply:
💯 	👍